modules/katello_devel/templates/install_rvm.sh.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+USER=<%= @user %>
+
+SERVERS=(
+    hkp://keys.gnupg.net
+    hkp://pgp.mit.edu
+)
+
+for server in "${SERVERS[@]}"
+do
+    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
```

Comment:
@ekohl opened an update here if you have time to look https://github.com/Katello/puppet-katello_devel/pull/37